x-pack/plugins/viz_editor/common/query_types.ts

```diff
@@ -62,44 +73,51 @@ export interface LitDate {
 
 type Literal = LitAtomic | LitDate;
 
-type ComparisonArg = Literal | Column;
+type ComparisonArg = Literal | ColumnOperation;
+
+export type BooleanOperator = 'and' | 'or' | '>' | '>=' | '<' | '<=' | '=' | '<>';
+
+export interface GenericBoolean {
```

Comment:
Similar to above, maybe this should be: `BooleanOperation`, though that gets confusing with `BooleanOperator`. Maybe we don't need this definition at all? It doesn't seem to be adding value at the moment. We can always add it in if / when we do need it.

Reply:
You're suggesting removing all the filtering via Where clauses? I agree it's not used yet, but I bet we will want it soon